src/shogun/evaluation/CrossValidation.cpp

```diff
@@ -331,7 +333,8 @@ float64_t CCrossValidation::evaluate_one_run(
 			SG_UNREF(labels);
 			SG_UNREF(evaluation_criterion);
 			SG_UNREF(result_labels);
-			SG_UNREF(fold)
+			SG_UNREF(true_labels);
+			SG_UNREF(fold);
```

Comment:
did you valgrind the updated code somehow to avoid memory leaks here?

Reply:
Yes, I did. Actually, I had to place those lines to fix a memory leak.